sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_receiver_async.py

```diff
@@ -316,6 +320,17 @@ def _from_connection_string(
         return cls(**constructor_args)
 
     def _create_handler(self, auth: Union["pyamqp_JWTTokenAuthAsync", "uamqp_JWTTokenAuthAsync"]) -> None:
+        link_properties = {CONSUMER_IDENTIFIER: self._name}
+        if self._session_id == NEXT_AVAILABLE_SESSION:
+            timeout_in_ms = self._max_wait_time * 1000 if self._max_wait_time else 0
+            # open_receive_link_base_jitter_in_ms = 100
+            # open_recieve_link_buffer_in_ms = 20
+            # open_receive_link_buffer_threshold_in_ms = 1000
+            # jitter_base_in_ms = min(timeout_in_ms * 0.01, open_receive_link_base_jitter_in_ms)
+            # timeout_in_ms = math.floor(timeout_in_ms - jitter_base_in_ms * random.random())
+            # if timeout_in_ms >= open_receive_link_buffer_threshold_in_ms:
+            #     timeout_in_ms -= open_recieve_link_buffer_in_ms
+            link_properties[OPERATION_TIMEOUT] = timeout_in_ms - 100
```

Comment:
would this entire section only apply to pyamqp? If so, I think we'd need to move this into the _pyamqp_transport.py file (and in there, get the max_wait_time by calling receiver._max_wait_time).

Reply:
oo yes it does, will move to transport file